pages/reference/base-images/base-images.md

```diff
@@ -108,92 +107,97 @@ CMD ["node", "main.js"]
 
 ### Supported Architectures, Distros and Languages
 
-Currently balenalib supports the following OS distributions and Language stacks, if you would like to see others added, create an issue on the [balena base images repo](https://github.com/balena-io-library/base-images/issues).
+Currently, balenalib supports the following OS distributions and Language stacks, if you would like to see others added, create an issue on the [balena base images repo]({{ $links.githubLibrary }}/base-images/issues).
+
+| Distribution | Default                      | Supported Architectures                      |
+| ------------ | ---------------------------- | -------------------------------------------- |
+| Debian       | Debian GNU/Linux 10 (buster) | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
+| Alpine       | Alpine Linux v3.10           | armv6, armv7hf, aarch64, amd64, i386         |
+| Ubuntu       | 18.04 LTS (bionic)           | armv7hf, aarch64, amd64, i386                |
+| Fedora       | Fedora 30                    | armv7hf, aarch64, amd64, i386                |
+
+| Language | Default | Supported Architectures                      |
+| -------- | ------- | -------------------------------------------- |
+| Node.js  | v13.5.0 | armv6, armv7hf, aarch64, amd64, i386         |
+| Python   | v2.7.17 | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
+| OpenJDK  | v11-jdk | armv7hf, aarch64, amd64, i386, armv6         |
+| Go       | 1.13.4  | armv7hf, aarch64, amd64, i386, armv6         |
+| Dotnet   | 3.1-sdk | armv7hf, aarch64, amd64                      |
```

Comment:
Some of the versions here have a `v` at the front and some don't? I guess that'll be fixed when this page is more auto-generated? :slightly_smiling_face:

Reply:
Thanks for these nits as good to clean up. I've removed `v` as clearly these shouldn't be specified in the tag and could lead to confusion.